samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/ServiceAreas.razor

```diff
@@ -1,8 +1,10 @@
 ﻿@page "/service-areas"
 <PageTitle>Service Areas</PageTitle>
 <h1>Service Areas</h1>
-<a target="_blank" href="https://developers.arcgis.com/javascript/latest/find-service-areas/">ArcGIS API for JavaScript</a>
-
+<a target="_blank" href="https://developers.arcgis.com/javascript/latest/find-service-areas/">ArcGIS API for JavaScript Reference</a>
+<p class="instructions">
+    Click on any point on the map to be shown a set of "Service Areas", drivable distances in 5, 10, and 15 minute bands.
```

Comment:
Can this be reworded? I'm not sure if Service Areas are drivable distances or if you can see service areas and drivable distances.

Reply:
@codergrl I'm not sure I understand. The definition of Service Area on the linked ArcGIS JS page is
> A service area, also known as an isochrone, is a polygon that represents the area that can be reached when driving or walking on a street network.

Reply:
I see. Ok, how about rewording it to something like this:

Click on any point on the map to be shown a set of "Service Areas". Service Areas are drivable distances in 5, 10, and 15 minute bands.

Reply:
sounds good!